pkg/virtual/framework/cmd/cmd.go

```diff
@@ -67,11 +70,15 @@ type SubCommandOptions interface {
 func APIServerCommand(out, errout io.Writer, stopCh <-chan struct{}, subCommandOptions SubCommandOptions) *cobra.Command {
 	options := &APIServerOptions{
 		Output:            out,
-		SecureServing:     kubeoptions.NewSecureServingOptions().WithLoopback(),
+		SecureServing:     kubeoptions.NewSecureServingOptions(),
 		Authentication:    genericapiserveroptions.NewDelegatingAuthenticationOptions(),
 		SubCommandOptions: subCommandOptions,
 	}
 
+	options.SecureServing.ServerCert.CertKey.CertFile = filepath.Join(".", ".kcp", "apiserver.crt")
+	options.SecureServing.ServerCert.CertKey.KeyFile = filepath.Join(".", ".kcp", "apiserver.key")
+	options.SecureServing.BindPort = SecurePortDefault
+	options.Authentication.SkipInClusterLookup = true
```

Comment:
we might want to start an options package here too, and move those default into options.NewSecureServing

Reply:
Could we leave that for a next refactoring PR ?

Reply:
Can we leave that for a broader refactoring task in a next PR ?

Reply:
Done after the rebase on your PR and the additional commits you provided

Reply:
sure, was meant as follow-up.